### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by mcallnisn**

### CODE REVIEW

**Comment by mcallnisn** [comment]
@fcwheat could you approve this change to upgrade the underlying dependencies?

File: test.js [hunk 1]
```diff
@@ -6,14 +6,15 @@ const redis = require('./index');
 const sleep = (ms) => new Promise((resolve) => setTimeout(resolve, ms));
 
 tap.test('createClient', async (t) => {
-  t.type(redis.createClient(), 'RedisClient');
+  let client = redis.createClient();
```
Comment (fcwheat) [inline test.js:9]: `const` would be preferred here since the client variable isn't being modified in this scope

File: package.json [hunk 2]
```diff
@@ -13,15 +13,15 @@
     "test": "tap test.js"
   },
   "dependencies": {
-    "async-redis": "^1.1.7",
-    "redis-mock": "^0.49.0",
+    "async-redis": "^2.0.0",
```
Comment (fcwheat) [inline package.json:16]: do you know if `async-redis` has any breaking changes that we should keep in mind?

File: test.js (hunk 1 above)
Comment (mcallnisn) [reply test.js]: fixed

File: package.json (hunk 2 above)
Comment (mcallnisn) [reply package.json:16]: according to the [docs](https://www.npmjs.com/package/async-redis/v/2.0.0):

> This library does very little modification to the api of node_redis. It simply appends a promise resolving/rejecting callback for every command.

Comment (fcwheat) [reply package.json:16]: i meant more about the major version change from `1.X.X` to `2.X.X`. i'd typically associate this with breaking changes or other large features that were introduced. i haven't found good documentation summarizing the update on the `async-redis` side yet though so trying to track that down.

Comment (fcwheat) [reply package.json:16]: overall [this](https://github.com/moaxaca/async-redis/compare/v1.1.7...2.0.0) is looking ok to me. lots of security upgrades, some typescript definitions, github actions stuff, and some newer command support

**Review by fcwheat — APPROVED** [review]

**Comment by mcallnisn** [comment]
@fcwheat thanks for the review and thumbs up, do you have the ability to merge and publish a new release?

**Comment by fcwheat** [comment]
> @fcwheat thanks for the review and thumbs up, do you have the ability to merge and publish a new release?

done

**Pull request merged**